features/all-positions/LiquidationActionDialog.tsx

```diff
@@ -212,7 +212,7 @@ const LiquidationActionDialog = (props: DialogProps) => {
         try {
           if (needCollateralAllowance()) await setMaxCollateralAllowance();
           const tx = await emp.dispute(
-            liquidatedPosition.sponsorPlusId,
+            liquidatedPosition.liquidationId,
```

Comment:
@chrismaree This was a bug in `dispute()` method, ID should be passed not the sponsorPlusId

Reply:
good catch, thanks for fixing!